packages/rum/test/e2e/standalone-html/app.e2e-spec.js

```diff
@@ -39,6 +39,19 @@ describe('standalone-html', function() {
     return allowSomeBrowserErrors(['timeout test error with a secret'])
   })
 
+  it('should run the async usecase', function() {
+    browser.url('/test/e2e/standalone-html/index-async.html')
+    browser.waitUntil(
+      () => {
+        return $('#test-element').getText() === 'Passed'
+      },
+      10000,
+      'expected element #test-element'
+    )
+
+    return allowSomeBrowserErrors(['timeout test error with a secret'])
```

Comment:
I would also test for page-load transaction and spans in this test, since we don't have any async test cases till now.